modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java

```diff
@@ -143,12 +143,15 @@ protected void configure() {
         }
 
         if (recordingMode != VncRecordingMode.SKIP) {
-            try {
-                vncRecordingDirectory = Files.createTempDirectory(TC_TEMP_DIR_PREFIX).toFile();
-            } catch (IOException e) {
-                // should never happen as per javadoc, since we use valid prefix
-                logger().error("Exception while trying to create temp directory " + vncRecordingDirectory.getAbsolutePath(), e);
-                throw new ContainerLaunchException("Exception while trying to create temp directory", e);
+
+            if(vncRecordingDirectory == null){
```

Comment:
```suggestion
            if (vncRecordingDirectory == null) {
```

Reply:
Oh yes, damm...
Do you have a formater for the project to import via IntelliJ?

Reply:
We use IDEA's standard format with some manual asks that are not easily configurable :)